kubernetes_platform/python/test/snapshot/data/toleration.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp import dsl
+from kfp import kubernetes
+from kubernetes.client import V1Toleration
+
+
+@dsl.component
+def comp():
+    pass
+
+
+@dsl.pipeline
+def my_pipeline():
```

Comment:
is `my_pipeline` a good name for this? I think giving a more meaningful name could help when searching maybe?

Reply:
This is just following suit with the other snapshots: https://github.com/kubeflow/pipelines/blob/master/kubernetes_platform/python/test/snapshot/data/node_selector.py#L25.